composer.json

```diff
@@ -15,11 +15,11 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "league/commonmark": "dev-master"
+        "league/commonmark": "0.7.*"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.3",
-        "codeclimate/php-test-reporter": "~0.1"
```

Comment:
Again, I disagree. 0.2 could be a breaking release. The maintainers are free to re-namespace for example, which would trash everything...